gallery/src/react-daterange-picker.d.ts

```diff
@@ -0,0 +1 @@
+declare module '@wojtekmaj/react-daterange-picker';
```

Comment:
This daterange picker lib does not have any typescript typings.  This line enables typescript to accept it without errors (though of course also without type safety)